docs/partition_ips.rst

```diff
@@ -0,0 +1,62 @@
+Partition IPs in Modin (experimental)
+=====================================
+
+If you are working with Modin DataFrame and would like to unwrap its remote partitions
+for your needs  (pass them to another function that will be processed on a concrete node of the cluster,
+for instance), you can use IPs of the remote partitions. In that case you can pass the partitions
+having needed IPs to your function. It can help with minimazing of data movement between nodes. However,
+it is worth noting that for Modin on ``Ray`` engine with ``pandas`` backend IPs of the remote partitions may not match
+actual locations if the partitions are lower than 100 kB. Ray saves such objects (<= 100 kB, by default) in in-process store
```

Comment:
I think we should insert link on Ray docpage with info about in-process store.

Reply:
We probably want a user to refer to the entire ray documentation.